src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="What is ETH?"
+                description="The currency of the Ethereum network is called ether or ETH."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.eth)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/eth/">What is ETH?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Where can I get ETH?"
+                description="There are many ways to aquire ETH depending on your location."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.impact)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/get-eth/">Where can I get ETH?</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/smart-contracts/">
+                What are smart contracts?
+              </DocLink>
+              <DocLink to="/developers/docs//intro-to-ethereum/">
+                A technical introduction to Ethereum
+              </DocLink>
+              <DocLink to="/web3/">What is Web3?</DocLink>
+              <DocLink
+                to="https://www.youtube.com/watch?v=WSN5BaCzsbo"
+                isExternal
+              >
+                Decentralizing Everything
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[1].id}>{tocItems[1].title}</h2>
+            <p>
+              It’s actually quite simple once you get the hang of it. You need
+              an application that is commonly called a “wallet”. It's an app
+              that helps you store your funds and to authenticate and interact
+              with applications on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is a wallet?"
+                description="How you manage your ETH and your Ethereum accounts."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.wallet)}
+                      alt="Illustration of a robot."
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/">What is a wallet?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Find a wallet"
+                description="We will help you find the right wallet based on the features you care about."
```

Comment:
Lighten the tone/fix a preposition at the end of the sentence 

```suggestion
                description="We'll help you find the best wallet based on the features that matter to you."
```